packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.test.ts

```diff
@@ -438,97 +438,6 @@ describe('user-storage/user-storage-controller - syncInternalAccountsWithUserSto
     expect(mockAPI.mockEndpointGetUserStorage.isDone()).toBe(false);
   });
 
-  it('returns void if syncing has recently been performed', async () => {
```

Comment:
Could we have new test(s) for the corresponding desired behavior?

Reply:
I'm not sure what you mean @legobeat , since I've removed the functionality that was tested here as well 😛
All the other cases are already covered by tests I've written before. LMK!